frontend/build-configs/bayern/index.ts

```diff
@@ -52,6 +52,34 @@ export const bayernCommon: CommonBuildConfigType = {
         boxDecorationRadius: 1,
     },
     iconInAboutTab: "assets/bayern/icon.png",
+    introSlide1: {
+        title: "Willkommen!",
+        description: "Vielen Dank, dass Sie sich die App zur " +
+            "Bayerischen Ehrenamtskarte heruntergeladen haben!",
+        pathName: "assets/bayern/icon.png"
+    },
+    introSlide2: {
+        title: "Wie kann ich eine Ehrenamtskarte beantragen?",
+        description: "In dem Formular geben Sie Informationen über Sie und Ihre " +
+            "ehrenamtliche Tätigkeit an.\nAnschließend wird " +
```

Comment:
Not sure about "sich" :D

```suggestion
        description: "Im Formular geben Sie Informationen über sich und Ihre " +
            "ehrenamtliche Tätigkeit an.\nAnschließend wird " +
```